CIScripts/Test/PesterHelpers/PesterHelpers.Tests.ps1

```diff
@@ -117,6 +117,5 @@ Describe "PesterHelpers" {
         Mock Get-Date {
             return $Script:MockStartDate.AddSeconds($Script:SecondsCounter)
         }
-        Mock Write-Host { return }
```

Comment:
I am not sure why this mock was removed.

Reply:
I guess somebody's realized it's not needed anymore.

Reply:
Correct, this mock was there for the dot printing if you remember @krdln